### ISSUE TITLE
conversion script 'ddl2cpp' does not support boolean type

### ISSUE THREAD

**Issue by mitsutaka-takeda** [rbock/sqlpp11#53]
When I try to convert the following SQL to C++ header, 
```sql
CREATE TABLE TEST (
  id varchar(255) PRIMARY KEY,
  flag boolean
) ENGINE=InnoDB DEFAULT CHARSET=utf8
```
, I got
```sh
Traceback (most recent call last):
  File "/home/mtakeda/src/cpp/test_sqlpp11/sqlpp11/scripts/ddl2cpp", line 159, in <module>
    traitslist = [NAMESPACE + '::' + types[sqlColumnType]];
KeyError: 'boolean'
```

The 'types' table in ddl2cpp has the key 'bool' instead of 'boolean'. Is this just a typo? I'm using the revision 'e152e29'.

**Comment by rbock** [comment]
`bool` is the type I use :-)

I'll add `boolean` as well.

**Comment by mitsutaka-takeda** [comment]
Thank you!

**Comment by rbock** [comment]
Fixed in release 0.35
